packages/thumbprint-react/components/Fab/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import styles from './index.module.scss';
+
+interface TextFabProps {
+    /**
+     * Children content to render.
+     */
+    children: React.ReactNode;
+    /**
+     * Visually highlights the fab if it‘s active.
+     */
+    active?: boolean;
+    /**
+     * Function that will run when the fab is clicked on.
+     */
+    onClick?: () => void;
+}
+
+export function TextFab({ children, active = false, onClick = noop }: TextFabProps): JSX.Element {
+    return (
+        <button
+            className={classNames(styles.root, styles.textFab, {
+                [styles.active]: active,
+                [styles.inactive]: !active,
+            })}
+            type="button"
+            onClick={onClick}
+        >
+            {children}
+        </button>
+    );
+}
+
+interface IconFabProps {
+    /**
+     * Icon from [Thumbprint Icons](/icons/) to render within the button. It must be one of the
+     * `small` icons.
+     */
+    icon: React.ReactNode;
+    /**
+     * Visually highlights the fab if it‘s active.
+     */
+    active?: boolean;
```

Comment:
Rather than `active` as a boolean, I think it'd be better to use `theme="primary"` and `theme="secondary"` as we put in the API review meeting notes.

A few benefits:

* Terminology is familiar since buttons also use these themes.
* If we wanted to add a third theme in the future, it'd be easy with an enum but difficult with a boolean.